src/composables/auth.ts

```diff
@@ -105,41 +126,41 @@ export const useAuth = createCustomScopedComposable(() => {
       return;
     }
     composableInitialized = true;
-    await checkSecureLoginAvailability();
+    await checkBiometricLoginAvailability();
   })();
 
   watch(
```

Comment:
Not sure if this is the block, that causing this, but I've noticed an issue - when you try to restore the wallet or create new one you are asked to set the password. But when you switch the tab to different one and come back again you are asked about unlocking the app with password :)
![image](https://github.com/user-attachments/assets/058e1fc0-3741-4bf9-be3d-55bdc5ad0808)

Reply:
Fixed within different PR and merged.